lib/python/Pyxsim/testers.py

```diff
@@ -60,9 +60,9 @@ def run(self, output):
         golden = self._golden
         regexp = self._regexp
         if isinstance(golden, list):
```

Comment:
you could argue that isinstance() is not very pythonic! But unrelated to this change.